src/components/AddBook/AddBook.js

```diff
@@ -18,35 +19,46 @@ const BookAdd = () => {
     return <p>Error occurred while fetching books</p>;
   }
   return (
-    <div className="book-wrap">
+    <div className="book-container">
       {books.map((book) => (
         <div className="books" key={book.item_id}>
-          <ul>
-            <li>
-              {book.category}
-              {' '}
-              <br />
-              {' '}
-              {book.title}
-              {' '}
-              By
-              {' '}
-              {book.author}
-              {' '}
-            </li>
-            <button
-              type="button"
-              className="remove"
-              onClick={() => {
-                dispatch(removeBook(book.item_id));
-              }}
-            >
-              Remove
-            </button>
+          <div className="book-add">
+            <h2 className="category">{book.category}</h2>
+            <h1 className="title">{book.title}</h1>
+            <h3 className="author">{book.author}</h3>
+            <div className="bttns">
+              <div className="bttn">Comments</div>
+              <div
+                className="bttn"
+                role="button"
+                tabIndex="0"
+                onClick={() => {
+                  dispatch(removeBook(book.item_id));
+                }}
+              >
+                Remove
+              </div>
+              <div className="bttn">Edit</div>
+            </div>
+          </div>
+          <div className="progress-bar">
+            <ProgressB />
+            <ul className="pro-text">
+              <li><h2>65%</h2></li>
+              <li><p>Completed</p></li>
+            </ul>
+          </div>
+          <div className="chapter-container">
+            <ul>
 
-          </ul>
+              <li><h2 className="current-h2">Current Chapter</h2></li>
+              <li><h3 className="chapter-h3">Chapter 17</h3></li>
+              <li><h4 className="update-h4">Update Progress</h4></li>
+            </ul>
+          </div>
```

Comment:
For this part too, kindly use the fonts provided in the design to make sure we stick to it:

Your design:
![Screenshot from 2023-04-22 19-51-01](https://user-images.githubusercontent.com/97834160/233799340-db294cc0-103b-4fc7-af79-493781e6fe25.png)

Zeplin:

![Screenshot from 2023-04-22 19-52-33](https://user-images.githubusercontent.com/97834160/233799357-1aff8ffd-0014-41c1-91e7-e7601ca58b5e.png)